### DISCUSSION TITLE
How to enable modules/extentions in php cloud functions?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#1490] [Q&A] [answered]
1. Is that possible to enable extentions of php for an example mongodb etc.
2. Is that possible to have custom php.ini
3. Can PECL extentions can be installed?

**Comment by eldadfux** [comment, marked as answer]
We don't offer something like that right now. You could try to use this fork and enable your custom runtime: . In the future we might include some popular extensions as part of the official runtime. 

https://github.com/appwrite/appwrite/pull/1284
